### ISSUE TITLE
Card Text sweep: Tiamat

### ISSUE THREAD

**Issue by TopherGerkey** [SotMSteamMods/CauldronMods#881]
Hydra instruction cards: Very minor graphical issue: the text boxes are white. instead of matching the colors on the physical cards.

More importantly, the flip side of Hydra's The Jaws of Winter should be The Frigid Earth, but instead it's showing a duplicate of The Noxious Fire. I think this is only a display issue but it's a big one.

Tiamat 2199 is missing an Advanced text box on her front side.
